src/components/badge/Badge.tsx

```diff
@@ -4,23 +4,35 @@ import { SBadgeIconContent } from "./styles/SBadgeIconContent";
 import { SBadgeIcon } from "./styles/SBadgeIcon";
 import { BadgePosition } from "./BadgePosition";
 export { BadgePosition } from "./BadgePosition";
```

Comment:
I think BadgePosition is a Enum - can we prefix it with a `E` so its `EBadgePosition` - that will prevent from miss use.